core/domain/question_domain_test.py

```diff
@@ -106,15 +106,15 @@ def test_create_default_question(self):
         question_id = 'col1.random'
         language_code = 'en'
         question = question_domain.Question.create_default_question(
-            question_id, language_code)
+            question_id, language_code).to_dict()
         default_question_data = exp_domain.State.create_default_state(
             feconf.DEFAULT_INIT_STATE_NAME, is_initial_state=True).to_dict()
 
-        self.assertEqual(question.question_id, question_id)
-        self.assertEqual(question.question_data_schema_version, 1)
+        self.assertEqual(question['question_id'], question_id)
+        self.assertEqual(question['question_data_schema_version'], 1)
         self.assertEqual(
-            question.question_data.to_dict(), default_question_data)
-        self.assertEqual(question.language_code, 'en')
+            question['question_data'].to_dict(), default_question_data)
```

Comment:
When question is changed to a dict, all its fields would also be changed to dict, so this would throw an error, if not, modify question.to_dict() to change question_data also to dict.
A dictionary should not have any class objects in it.

Reply:
done